eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -301,10 +306,18 @@ stages:
               - ${{ each artifact in parameters.Artifacts }}:
                 - $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo/${{artifact.name}}.json
             WorkingDirectory: $(System.DefaultWorkingDirectory)
-            TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
+            TargetDocRepoOwner: Azure
             TargetDocRepoName: ${{parameters.TargetDocRepoName}}
             Language: 'dotnet'
             DailyDocsBuild: true
             SparseCheckoutPaths:
               - /api/overview/azure/
               - /metadata/
+        - ${{ each artifact in parameters.Artifacts }}:
+          - ${{if ne(artifact.skipPublishDocMs, 'true')}}:
+            - template: /eng/pipelines/templates/steps/fetch-namespaces.yml
+              parameters:
+                DotnetNupkgLocation: "$(Pipeline.Workspace)/${{parameters.ArtifactName}}"
+                ArtifactName: ${{artifact.name}}
+                TargetBranch: $(TargetBranchName)
```

Comment:
Where are we setting the TargetBranchName?

Reply:
It is right in update-docsms-metadata.yml